
_SYS_LIMITS_H is used by our /usr/include/sys/limits.h so we need to use
a different guard for the sys-limits.h that ships with coreutils.

diff -wpruN '--exclude=*.orig' coreutils-8.30~/lib/sys-limits.h coreutils-8.30/lib/sys-limits.h
--- coreutils-8.30~/lib/sys-limits.h	2018-06-24 04:56:28.000000000 +0000
+++ coreutils-8.30/lib/sys-limits.h	2018-07-09 09:33:09.648022168 +0000
@@ -15,8 +15,8 @@
    You should have received a copy of the GNU General Public License
    along with this program; if not, see <https://www.gnu.org/licenses/>.  */
 
-#ifndef _SYS_LIMITS_H
-#define _SYS_LIMITS_H
+#ifndef _COREUTILS_SYS_LIMITS_H
+#define _COREUTILS_SYS_LIMITS_H
 
 #include <limits.h>
 
